app/routes/document-configure-response.js

```diff
@@ -24,6 +25,8 @@ module.exports = [{
     auth: { scope: [admin] },
     handler: async (request, h) => {
       const documentId = request.payload.documentId
+      setInSession(request, 'knowledge', request.payload.knowledge || [])
```

Comment:
I think it might be a good idea to move session keys out into a constants folder. Also we should be doing null checks somewhere central.

Take a look here for inspiration:
https://github.com/DEFRA/aphw-ddi-portal/tree/main/app/session/cdo

Reply:
I've moved the session code